lib/tasks/populate_postcodes_table.rake

```diff
@@ -0,0 +1,26 @@
+require "json"
+
+desc ""
+task :populate_postcodes_table do
+  CSV.open("./postcodes.csv", "wb") do |csv|
```

Comment:
It would be nice to make this a user argument ([example](https://github.com/alphagov/whitehall/blob/126891bf9f2ff49877c5649aa069ca21575b23f6/lib/tasks/publishing_api.rake#L281-L291)), so that the dev can specify the absolute path to the CSV.